### DISCUSSION TITLE
Linearization explanation

### DISCUSSION THREAD

**Discussion by joeb8799** [NatLabRockies/ROSCO#143] [Q&A]
Hello,
I'm currently working on example 12 for my internship and i need to understand the concept of linearization used in example 12.
What are we linearizing and how can i generate the linearization files (.lin) from OPENFAST so that i can use these files for a turbine model that i have.

**Comment by dzalkind** [comment]
Example 12 uses a linearized turbine model from OpenFAST.  There are a lot of discussions on the web about linearizing OpenFAST and this might be a good start: https://github.com/OpenFAST/openfast/issues/738
